.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,48 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item ]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
```

Comment:
Include specific CUJs / use cases, actual examples of questions customers want  to answer using the data, and example sample queries.

Reply:
```suggestion
Describe the problem, issue, use case, or opportunity that this work item addresses.  Please include any examples of questions being asked by practitioners.
```

Reply:
We committed an alternate suggestion to this.  Resolving.